modules/core/src/main/scala/Theme.scala

```diff
@@ -0,0 +1,175 @@
+    def selectedManyInactive: String
+    def canceled: String
+    def error: String
+  end extension
+end Theme
+
+object Theme:
+  import TextFormatting.*
+
+  val Default: Theme = new Theme:
+
+    extension (s: String)
+      def prompt: String               = s.cyan
+      def emphasis: String             = s.bold
+      def input: String                = s
+      def option: String               = s.bold
+      def optionMany: String           = s
+      def noMatches: String            = s.bold
+      def nothingSelected: String      = s.underline
+      def selected: String             = s.green
+      def selectedMany: String         = s.underline.green
+      def selectedManyInactive: String = s.underline
+      def canceled: String             = s.red
+      def error: String                = s.red
+    end extension
+  end Default
+
+  val NoColors: Theme = new Theme:
+    extension (s: String)
+      def prompt: String               = s
+      def emphasis: String             = s
+      def input: String                = s
+      def option: String               = s
+      def optionMany: String           = s
+      def noMatches: String            = s
+      def nothingSelected: String      = s
+      def selected: String             = s
+      def selectedMany: String         = s
+      def selectedManyInactive: String = s
+      def canceled: String             = s
+      def error: String                = s
+    end extension
+  end NoColors
+
+  def Solarized: Theme = new Theme:
+    extension (s: String)
+      def prompt: String               = s.cyan
+      def emphasis: String             = s.bold.yellow
+      def input: String                = s
+      def option: String               = s.bold
+      def optionMany: String           = s
+      def noMatches: String            = s.lightBlue
+      def nothingSelected: String      = s.underline.lightGray
+      def selected: String             = s.lightGreen
+      def selectedMany: String         = s.bold.lightGreen
+      def selectedManyInactive: String = s.underline.lightGray
+      def canceled: String             = s.lightRed
+      def error: String                = s.red
+    end extension
+  end Solarized
+
+  def Monokai: Theme = new Theme:
+    extension (s: String)
+      def prompt: String               = s.green
+      def emphasis: String             = s.bold.magenta
+      def input: String                = s
+      def option: String               = s.bold
+      def optionMany: String           = s
+      def noMatches: String            = s.yellow
+      def nothingSelected: String      = s.underline.darkGray
+      def selected: String             = s.lightCyan
+      def selectedMany: String         = s.bold.lightCyan
+      def selectedManyInactive: String = s.underline.lightGray
+      def canceled: String             = s.red
+      def error: String                = s.bold.red
+    end extension
+  end Monokai
+
+  def Darcula: Theme = new Theme:
```

Comment:
I think these can just be static objects, `case object Darcula extends Theme:`

Reply:
Actually I tested a few of the themes, and to my eye they have nothing to do with respective known themes.

As such, they should be removed or re-done from scratch. The Default and NoColors can remain.